uniffi_macros/src/export.rs

```diff
@@ -117,7 +117,10 @@ fn fn_type_assertions(sig: &syn::Signature) -> TokenStream {
 
     let input_types = sig.inputs.iter().filter_map(|input| match input {
         syn::FnArg::Receiver(_) => None,
-        syn::FnArg::Typed(pat_ty) => Some(&pat_ty.ty),
+        syn::FnArg::Typed(pat_ty) => match &*pat_ty.pat {
+            syn::Pat::Ident(i) if i.ident == "self" => None,
```

Comment:
How about a comment here about why we don't need a type assertion for `self` params.  (Or should this be eventually implemented?).

Reply:
Hm, I think a type assertion might be generated for the whole impl block's self type. I will have to look at the code again.

Reply:
Oh, actually we don't have the type assertions at all in impl blocks / methods 😅